### ISSUE TITLE
Default port message

### ISSUE THREAD

**Issue by gonzalo-roberto-diaz** [aaronshaf/dynamodb-admin#42]
If one just types `npm start`(after installing with the current master), without setting any environment variable, the following message appears

```
DYNAMO_ENDPOINT is not defined (using default of http://localhost:8000)
listening on http://0.0.0.0:54180
```

This seems inconsistent. The actual, working port is 54180, 8000 is not served.

**Comment by rchl** [comment]
DYNAMO_ENDPOINT is the database endpoint to which `dynamodb-admin` is connecting in this case.

I guess the wording could use some improvement to make it clearer. Maybe it wouldn't have to mention 'http' protocol as that makes people think that it's a server that handles http requests while it's not (dynamodb works over http).

**Comment by gonzalo-roberto-diaz** [comment]
OK, but what I am saying is, when the `DYNAMO_ENDPOINT` system variable is not defined, the random port is the one being used, not the default 8000).

**Comment by rchl** [comment]
Again, the port on which dynamodb-admin is running has nothing to do with DYNAMO_ENDPOINT and its port. These are two different ports - one is where dynamodb-admin is exposed on and the other is where dynamodb (database) is running on. Dynamodb-admin doesn't run the database itself so it has to connect to one that user has running already. That is the endpoint to which the first line refers to. The second line is the endpoint on which dynamodb-admin is exposed.

Basically first line is the endpoint of the database and second line is the endpoint of the UI.
And looking at the code, I don't think we have a bug that the endpoint of the database gets random port assigned.

**Comment by rchl** [comment]
Someone just [fixed a bug ](#46) that made default port of the UI get randomly assigned instead of using 8001. Maybe that is the bug that you meant?

**Comment by rchl** [comment]
I've [updated slightly](1578feb7fb4698f7ae1c4f70bb8c603084282957) console output on start to maybe make it more clear.

Otherwise I don't think there is any bug here so closing.
